src/components/teaser-paid/teaser-paid.scss

```
@@ -0,0 +1,33 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  // Use !important here to not interfere with Firefox focus ring definition
+  // which appears in normalize css of several frameworks.
+  outline: none !important;
+
+  @include sbb.panel-variables;
+
+  --sbb-teaser-paid-chip-offset: var(--sbb-spacing-responsive-xxxs);
```

with this change a consumer would have chance to have different for top and left
```suggestion
  --sbb-teaser-paid-chip-inset: var(--sbb-spacing-responsive-xxxs) 0 0 var(--sbb-spacing-responsive-xxxs);
```
assignment then would just be `inset:  var(--sbb-teaser-paid-chip-inset)`

---

This was my first approach too, but if I use `0 0` it stretches to the bottom and to the right, and if I use `unset unset` it just doesn't work

---

Apparently if works with `var(--sbb-spacing-responsive-xxxs) auto auto var(--sbb-spacing-responsive-xxxs)`

---

Ok I see. than maybe we could just provide two vars for each? second one pointing to the first one.

---

Ah cool :) auto auto would be ok I think